index.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>My Favorite Food Recipe</title>
+    <link rel="dns-prefetch" href="//fonts.googleapis.com" />
+    <link rel="stylesheet" href="/Users/Mac/Documents/CYF/RecipeCoursework/style.css"/>
+  </head>
+
+  <body>
+   
+    <header>
+        <div class="text-area">
+            <h1>Traditional Ukrainian<br> mushroom soup</h1>
+            <hr>
+            <h3>Ancient Recipe From the Capathians</h3>
+        </div>  
+            <img class="background-img" src="/Users/Mac/Documents/CYF/RecipeCoursework/images/Healthy_Musroom_Veggie_Soup-12-2.jpeg"/>
+            <img src="/Users/Mac/Documents/CYF/RecipeCoursework/images/Healthy_Musroom_Veggie_Soup-12-2.jpeg"/>   
+    </header>
+    <br>
+    <br>
+
+    <main>
+        <div class="recipe-main">
+            <p>Today I would love to share with you my favorite Ukrainian mushroom soup recipe. </p>
+            <p>This Ukrainian mushroom soup recipe has surprisingly simple ingredients and
+            is easy to make. Despite the fact that it is considered a gourmet dish, even an inexperienced cook can make it without any special knowledge and effort.</p>
+            <p>In Ukraine, due to the diversity of Carpathian cuisine, people prepare this mushroom soup differently. Even in the same village, the taste of the soup can differ from house to house. In fact, in Ukraine , we say that the most delicious mushroom soup is always prepared with porcini mushrooms. They can be fresh or dried, however, dried mushrooms enhance the flavor and bring a more unique aroma.</p>
+            <p>If you don’t have porcini, use oyster mushrooms, chanterelle, or king trumpet mushrooms. But do not use buttons, portabello, or cremini ones. At least not for this soup.</p>
+            <br>
+            <hr>
+            <br>
+            <h3>How to Make Ukrainian Porcini Mushroom Soup</h3>
+            <div class="ingredients">
+                <img src="/Users/Mac/Documents/CYF/RecipeCoursework/images/2021_11_03_easy_cream_of_mushroom_soup_3.jpeg" alt="ingredients for mushroom soup"/>
+                <div class="ingredients-text">
+                    <h4>Ingridients</h4>
+                    <ul>
+                        <li>150 grams (5.5 oz) of dried porcini mushrooms </li><br>
+                        <li>1 Medium Carrot</li><br>
+                        <li>2-3 medium bay leaves </li><br>
+                        <li>1 big (or 2 smaller) onions</li><br>
+                        <li>1 small potato </li><br>
+                        <li>4 Tbsp of olive oil</li><br>
+                        <li>3 liters (12.5 cups) of water</li><br>
+                        <li>salt & ground black pepper to taste </li><br>
+                        <li>1/4 tsp of dill (fresh or frozen)</li><br>
+                        <li>sour cream (optional)</li><br>
```

Comment:
we should never do padding on the bottom of `<li>`

Instead use css
```css
li {
  padding: 0.2em
}
```

Reply:
@JarrodFolinoFC thanks for your comment, do you men <br>?